wolox-books/src/app/screens/Login/components/login/login.component.html

```diff
@@ -0,0 +1,17 @@
+<form [formGroup]="loginForm" class="form-container" (ngSubmit)="onSubmit()">
+  <div class="form-elements">
+    <img src="assets/w-logo.png" alt="Wolox logo" class="w-logo">
+    <label class="data-label">Email</label>
+    <input class="data-input" type="email" formControlName="email">
+    <span class="error-message" *ngIf="loginForm.get('email').touched">
+      {{ validator.errorMessage('email', loginForm) }}
+    </span>
+    <label class="data-label">Password</label>
+    <input class="data-input" type="password" formControlName="password">
+    <span class="error-message" *ngIf="loginForm.get('password').touched">
+      {{ validator.errorMessage('password', loginForm) }}
+    </span>
+    <button class="sign-up-btn" type="submit" [disabled]="loginForm.invalid">Login</button>
+  </div>
+  <button class="login-btn" type="button" [routerLink]="['/sign-up']">Sign Up</button>
```

Comment:
For routerLinks paths, I would use a constants file with the strings, In that way if you are using a path in different parts in the future, you will have to change only the constant.